cmd/operator-sdk/build/cmd.go

```diff
@@ -94,18 +95,38 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
 		return fmt.Errorf("command %s requires exactly one argument", cmd.CommandPath())
 	}
+	image := args[0]
+
+	// If the project has the new kubebuilder layout
+	if kbutil.HasProjectFile() {
+		//makeDockerBuild := "make docker-build IMG="+image
+		cmd := exec.Command("make", "docker-build", "IMG="+image)
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			return err
+		}
+		fmt.Printf("%s\n", out)
```

Comment:
Use `projutil.ExecCmd()` here so that output streams instead of buffers.

Reply:
Actually, didn't we say that we want to have this command basically do the same thing as the legacy build except that:
1. the Dockerfile is in a different location
2. the Dockerfile is a multistage build, so there's no need to build the binary in advance.